gui/gui/src/TRootCanvas.cxx

```diff
@@ -840,6 +841,17 @@ Bool_t TRootCanvas::ProcessMessage(Longptr_t msg, Longptr_t parm1, Longptr_t)
                         if (!fi.fFilename) return kTRUE;
                         dir = fi.fIniDir;
                         new TFile(fi.fFilename, "update");
```

Comment:
Probably here one should use `TFile::Open()` - it is more robust

Reply:
done

Reply:
Thanks!